js/index.js

```diff
@@ -64,4 +66,48 @@ const messageList = messageSection.querySelector('ul');
     messageList.appendChild(newMessage);
     messageSection.hidden = false;
     messageForm.item(0).reset();
-});
\ No newline at end of file
+});
+
+const githubRequest = new XMLHttpRequest();
+githubRequest.open('GET', 'https://api.github.com/users/raissanfg/repos');
+githubRequest.send();
+githubRequest.addEventListener('load', function() {
+const repositories = JSON.parse(this.response)
+
+const projectSection = document.querySelector('#projects')
+const projectList = projectSection.querySelector('ul')
+
+ for (let i=0; i < repositories.length; i++) {
+    var project = document.createElement('li');
+    var repositoryLink = document.createElement('a');
+    repositoryLink.href = repositories[i].html_url;
+    repositoryLink.innerHTML = repositories[i].name;
+    projectList.appendChild(project);
+    project.appendChild(repositoryLink);
+
+ }
+});
+
+function renderProjectsWithFetch() {
+    fetch('http://api.github.com/user/raissanfg/repos')
```

Comment:
There's a typo in the URL here; it should be the same as the one used in the XMLHttpRequest bit:
```suggestion
    fetch('https://api.github.com/users/raissanfg/repos')
```